index.js

```diff
@@ -35,6 +58,61 @@ app.get('/events/:id', function (req, res) {
 	events.getById(req, res, db);
 });
 
+app.get('/users', function (req, res) {
+    users.getAll(req, res, db);
+});
+
+app.get('/users/:id', function (req, res) {
+    users.getById(req, res, db);
+});
+
+// testing function to make sure that json is being sent to he url
+/*app.get('/testUserCreation', function (req, res) {
+    request({
+        method: 'POST',
+        url: 'http://localhost:8080/createUser',
+        json: { first: 'Bob', last: 'Builder', email: 'bob@builder.com' }
+    }, function (err, response, body) {
+        console.log('testing ' + body);
+    });
+});*/
+
+app.post('/createUser', function (req, res) {
```

Comment:
the endpoint for this should be /user  
"For example, for the URI http://example.com/customers, POST requests might be expected to represent new customers, each including their name, address, contact details and so on. " (https://en.wikipedia.org/wiki/POST_(HTTP))